circle.yml

```diff
@@ -76,6 +76,25 @@ jobs:
       - restore-gradle-cache
       # --stacktrace is just a temporary flag to help with debugging of ConcurrentModificationException that happens occasionally
       - run: ./gradlew testDebugUnitTest --stacktrace
+      - run:
+          name: Save test results
+          command: |
+            mkdir -p ~/junit/
+            find . -type f -regex ".*/build/test-results/.*xml" -exec cp {} ~/junit/ \;
+          when: always
+      - store_test_results:
+          path: ~/junit
+      - run:
+          name: Save test reports
+          command: |
+            mkdir -p ~/reports/
+            cp -R ./feature_album/build/reports/* ~/reports/feature_album
+            cp -R ./feature_favourite/build/reports/* ~/reports/feature_favourite
+            cp -R ./feature_profile/build/reports/* ~/reports/feature_profile
```

Comment:
The issue with this approach is that after adding a new module, we have to remember to about saving these reports. It would be much better to have generic command that could list all features (or folders containing `/build/reports/*` subfolder) and copy all these reports